spharpy/spherical.py

```diff
@@ -1,6 +1,431 @@
 import numpy as np
 import scipy.special as special
 import spharpy.special as _special
+import logging as logger
+from pyfar import Coordinates
+from functools import lru_cache
+from spharpy.samplings.helpers import calculate_sampling_weights
+
+
+class SphHarm:
+    def __init__(self, n_max, coords, basis_type='complex', channel_convention='acn', inv_transform_type=None,
+                 normalization='n3d'):
+        r"""
```

Comment:
We typically move the docstring before the `__init__`. This way it shows up as the class documentation and we do not need to include `__init__` in the documentation, which might be looking weird for users with little Python experience.